modules/form/src/components/adapters.tsx

```diff
@@ -128,6 +128,26 @@ export const TextFieldAdapter = (props:FieldRenderProps<any> ): ReactElement =>
     );
 };
 
+export const ScopeFieldAdapter = (props:FieldRenderProps<any> ): ReactElement => {
+
+    const { childFieldProps, input } = props;
+
+    return (
+        <>
```

Comment:
Redundant fragment. Fragments are only needed if there is no parent element.

✅ Needed here since there is no parent div.

```jsx
return (
     <div>Some div</div>
     <Scopes
          defaultValue={ childFieldProps?.defaultValue }
          label = { childFieldProps?.label }
          value = { childFieldProps?.value }
          required = { childFieldProps?.required }
          onChange = { input?.onChange }
          onBlur = { input?.onBlur }
          placeholder = { childFieldProps?.placeholder }
      />
)
```

❌ Not needed here since the return is wrapped in a parent element.

```jsx
return (
    <div>
       <div>Some div</div>
       <Scopes
            defaultValue={ childFieldProps?.defaultValue }
            label = { childFieldProps?.label }
            value = { childFieldProps?.value }
            required = { childFieldProps?.required }
            onChange = { input?.onChange }
            onBlur = { input?.onBlur }
            placeholder = { childFieldProps?.placeholder }
        />
    </div>
)
```

Reply:
Fixed with 51ded5bd3ca56d4ec2c9f85a6b73b30573767c72